api/disruption_types_test.go

```diff
@@ -66,47 +70,66 @@ var _ = Describe("DisruptionStatus.RemoveDeadTargets Test", func() {
 				matchingTargets = matchingTargets[:randCount]
 			})
 
-			It("expects status.Targets to be limited to alive targets", func() {
+			It("expects status.TargetInjections to be limited to alive targets", func() {
 				status.RemoveDeadTargets(matchingTargets)
-				Expect(status.Targets).To(HaveLen(randCount))
-				Expect(status.Targets).To(Equal(matchingTargets))
+				Expect(status.TargetInjections).To(HaveLen(randCount))
+				targetNames := status.TargetInjections.GetTargetNames()
+				sort.Strings(targetNames)
+				sort.Strings(matchingTargets)
+				Expect(targetNames).Should(BeEquivalentTo(matchingTargets))
 			})
 		})
 	})
 
 	When("matchingTarget has no element", func() {
 		BeforeEach(func() {
 			matchingTargets = []string{}
-			Expect(status.Targets).To(HaveLen(5))
+			Expect(status.TargetInjections).To(HaveLen(5))
 		})
 
-		It("expects status.Targets to be empty", func() {
+		It("expects status.TargetInjection to be empty", func() {
 			status.RemoveDeadTargets(matchingTargets)
-			Expect(status.Targets).To(HaveLen(0))
+			Expect(status.TargetInjections).To(HaveLen(0))
 		})
 	})
 
 	When("matchingTarget has all different elements", func() {
 		BeforeEach(func() {
-			matchingTargets = []string{"5.5.5.5", "6.6.6.6", "7.7.7.7", "8.8.8.8", "9.9.9.9"}
+			matchingTargets = []string{
+				"target-6",
+				"target-7",
+				"target-8",
+				"target-9",
+				"target-10",
+			}
 		})
 
-		It("expects status.Targets to be empty", func() {
+		It("expects status.TargetInjection to be empty", func() {
 			status.RemoveDeadTargets(matchingTargets)
-			Expect(status.Targets).To(HaveLen(0))
+			Expect(status.TargetInjections).To(HaveLen(0))
 		})
 	})
 })
 
+func makeValidTargetInjections() v1beta1.TargetInjections {
+	return v1beta1.TargetInjections{
+		"target-1": {},
+		"target-2": {},
+		"target-3": {},
+		"target-4": {},
+		"target-5": {},
```

Comment:
consider adding an entry with valid value. It helps code-reader understand the logic more thoroughly 🤗 .

Reply:
I am agree with you! I will add a better context 👍